nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/BigQueryAttributes.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.processors.gcp.bigquery;
+
+/**
+ * Attributes associated with the BigQuery processors
+ */
+public class BigQueryAttributes {
+    private BigQueryAttributes() {}
+
+    public static final String DATASET_ATTR = "bq.dataset";
```

Comment:
Prefer if we could prefix it with `gcp`. Example: `gcp.bq.dataset`. Thoughts?

Reply:
That sounds okay to me, I'll try and cover that in the rebase.

Reply:
@zenfenan After looking at this, that doesn't seem the match the defaults used by the other gcp processors. i.e. `gcs.bucket`.

Reply:
It's purely a nitpick. Feel free to ignore. I just wanted to follow the same approach that many cloud processor bundles use.